### DISCUSSION TITLE
Analysis Arena Submission: [Your Title Here]

### DISCUSSION THREAD

**Discussion by NiketaSengar** [Dose-of-Data/analysis-arena#23] [Beginner]
### Analysis Category

Beginner Entry

### Discord Community

Dose of Data

### GitHub Repository and Notebook Link

https://github.com/NiketaSengar/pokemon-analysis/blob/main/pokemon%20analysis%20(1).ipynb

### Formatting Guidelines Confirmation

Yes, I confirm that my submission follows the formatting guidelines, with no extraneous debug/print statements and all solutions and visuals are rendered.

### Libraries and Setup Directions (Optional)

-  numpy
-  pandas
- matplotlib
- seaborn

**Comment by dyerrington** [comment]
This is my overall feedback. Good job on this notebook. The most significant improvement I would make is to start with a set of explanations for what you found, why they are important, then explain each of the steps and why they are essential and related to the problem, and then conclude with a summary of your most important points and explain what you found and why.  

For example, there was a good exploration of how legendary vs non-legendary existed in various features. However, how would you explain to someone who needed to learn more about Pokemon how to differentiate them based on features and characteristics alone? It sThere's some overlap with regular Pokemon that have similar stats -- how would you describe the challenge of determining them?

Overall, good work 👍 

| Criteria                 | Feedback                                                                                     | Points |
|--------------------------|----------------------------------------------------------------------------------------------|--------------------|
| Understanding of Data    | Good initial understanding of the dataset. Include the total number of Pokémon and attribute summary for better understanding. | 10                 |
| Analysis Technique       | Basic EDA techniques used. Include more detailed visualizations and statistical summaries.    | 15                 |
| Quality of Insights      | Initial findings are present. More in-depth insights are needed, such as patterns or correlations between attributes. | 10                 |
| Accuracy                 | Interpretation of missing values is accurate and presented.                          | 5                  |
| Communication            | The Notebook is clear but could benefit from more explanations and structured sections.           | 7                  |
| Visualization            | Include more varied and detailed plots such as histograms, bar charts, and heatmaps.          | 5                  |
| Feedback Participation   |  | N/A                |
